core/src/main/java/com/googlecode/psiprobe/AbstractTomcatContainer.java

```diff
@@ -68,6 +69,9 @@ public boolean installContext(String contextName) throws Exception {
     return findContext(contextName) != null;
   }
 
+  /* (non-Javadoc)
```

Comment:
Yeah, it cleared more than 100 Javadoc warnings, but it also introduced more than 100 line length warnings. At least those are easier to fix.